src/index.html

```diff
@@ -4,9 +4,376 @@
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+         src="./images/logo.png"
+         alt="LOGO"
+         class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+             href="#apple"
+             class="nav__link is-active"
+            >
+             Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+             href="#samsung"
+             class="nav__link"
+            >
+             Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+             href="#smartphones"
+             class="nav__link"
+            >
+             Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+             href="#laptops&computers"
+             data-qa="nav-hover"
+             class="nav__link"
+            >
+             Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+             href="#gadgets"
+             class="nav__link"
+            >
+             Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+             href="#tablets"
+             class="nav__link"
+            >
+             Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+             href="#photo"
+             class="nav__link"
+            >
+             Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+             href="#video"
+             class="nav__link"
+            >
+             Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="container">
+        <div class="card" data-qa="card">
+          <img
+           src="images/imac.jpeg"
+           alt="imac"
+           class="card__image"
+          >
+
+          <h2 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__feedback">
+            <div class="card__stars">
+              <div class="card__stars-img"></div>
+              <div class="card__stars-img"></div>
+              <div class="card__stars-img"></div>
+              <div class="card__stars-img"></div>
+              <div class="card__stars-img"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+
+
+          <div class="card__price-container">
+            <p class="card__price-label">Price:</p>
+            <p class="card__price">$2,199</p>
+          </div>
+
+          <a
+              href="#"
+              class="card__buy"
+              data-qa="card-hover"
+            >
+              Buy
+          </a>
```

Comment:
Attributes should only have 2 spaces. You need to check and fix it everywhere
```suggestion
          <a
            href="#"
            class="card__buy"
            data-qa="card-hover"
          >
            Buy
          </a>
```